src/wallet/scriptpubkeyman.cpp

```diff
@@ -553,3 +553,24 @@ void LegacyScriptPubKeyMan::DeriveNewChildKey(WalletBatch &batch, CKeyMetadata&
     if (!batch.WriteHDChain(hdChain))
         throw std::runtime_error(std::string(__func__) + ": Writing HD chain model failed");
 }
+
```

Comment:
In commit "Implement LoadKeyPool in LegacyScriptPubKeyMan" (0d701bd42d0d9db9da858a4e6ae346ae0b1dda90)

Empty `src/qt/stqG2SM` file in this commit should be dropped

Reply:
How'd that get there? Removed it.